pig/src/main/java/org/apache/iceberg/pig/IcebergPigInputFormat.java

```diff
@@ -68,6 +72,7 @@ public class IcebergPigInputFormat<T> extends InputFormat<Void, T> {
   private List<InputSplit> splits;
 
   IcebergPigInputFormat(Table table, String signature) {
+    LOG.warn("Iceberg Pig is now deprecated. It will be removed in Iceberg 1.8.");
```

Comment:
if we really want to keep the log message, then I'd slightly update it to `Iceberg Pig is deprecated and will be removed in Iceberg 1.8.0`

Reply:
It works for me 😄 

I'm updating the PR accordingly .